nautobot/core/tests/test_jobs.py

```diff
@@ -70,7 +70,7 @@ def test_export_with_constrained_permission(self):
             username=self.user.username,  # otherwise run_job_for_testing defaults to a superuser account
             content_type=ContentType.objects.get_for_model(Status).pk,
         )
-        self.assertEqual(job_result.status, JobResultStatusChoices.STATUS_SUCCESS)
+        self.assertJobResultStatus(job_result, JobResultStatusChoices.STATUS_SUCCESS)
```

Comment:
Probably only need for this and below right? Since the default parameter is already `JobResultStatusChoices.STATUS_SUCCESS`
```suggestion
        self.assertJobResultStatus(job_result)
```

Reply:
Yeah, I got lazy here. It's harmless but redundant; I can remove it if you think it's worth the effort.